test/doc/from_cylindrical.cpp

```diff
@@ -0,0 +1,22 @@
+#include <kyosu/kyosu.hpp>
```

Comment:
why this AND conversions.cpp ?

Reply:
I have written all unit test, but I still have to write the doc tests. This is probably old history.